source/site_ops/how-tos/matamo-analytics.rst

```diff
@@ -0,0 +1,49 @@
+Configuring Analytics
```

Comment:
I think this title is misleading. When I saw it in the TOC, I thought it was about enabling Aspects and was confused why I couldn't find the doc on Matamo. The file name and the document title should match. So this should be called "Matamo Analytics", or it could be called "Configuring Site-wide Analytics Tools" and the doc title made to match.

Reply:
I agree @sarina I was also a bit on the fence about the title so thanks a lot for your suggestion I have named is a site-wide analytics tool. This makes more sense.